sdk/ai/azure-ai-generative/azure/ai/generative/index/_mlindex.py

```diff
@@ -212,24 +221,25 @@ def as_langchain_vectorstore(self, credential: Optional[TokenCredential] = None)
                         index_name=self.index_config.get("index"),
                         embedding_function=self.get_langchain_embeddings(credential=credential).embed_query,
                         search_type="hybrid",
-                        semantic_configuration_name=self.index_config.get("semantic_configuration_name", "azureml-default"),
+                        semantic_configuration_name=self.index_config.get(
+                            "semantic_configuration_name", "azureml-default"
+                        ),
                         user_agent=f"azureml-rag=={version}/mlindex,langchain=={langchain_version}",
                     )
```

Comment:
@tot0, Do you think we need to have a different user_agent string? i.e. "azure-ai-generative-index ==..."?